src/index.js

```diff
@@ -54,7 +55,17 @@ export default function typescript ( options = {} ) {
 		throw new Error( `rollup-plugin-typescript: Couldn't process compiler options` );
 	}
 
-	const compilerOptions = parsed.options;
+	// let typescript load inheritance chain if there are base configs
+	const extendedConfig = !tsConfig.extends ?
+		null :
+		typescript.parseJsonConfigFileContent(tsConfig, typescript.sys, process.cwd(), parsed.options);
+	if (extendedConfig && extendedConfig.errors.length) {
+		extendedConfig.errors.forEach( error => console.error( `rollup-plugin-typescript: ${ error.messageText }` ) );
```

Comment:
we generally want to shy away from our own console output here. instead, use [`this.warn`](thiswarnwarning-string--rollupwarning-position-number---column-number-line-number---void)

Reply:
This plugin is currently using console.error rather than this.warn, but we're switching it for the migration. You can choose to switch it now or I can switch it later when migrating 🙂